bindings/azure/cosmosdb/cosmosdb.go

```diff
@@ -189,3 +212,18 @@ func (c *CosmosDB) lookup(m map[string]interface{}, ks []string) (val interface{
 
 	return c.lookup(m, ks[1:])
 }
+
+func isTooManyRequestsError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	var requestError *documentdb.RequestError
+	if errors.As(err, requestError) {
+		if requestError.Code == "429" {
```

Comment:
Please make it as const.

Reply:
Or use a constant from the library if one is exposed.

Reply:
> > > I successfully reproduced the problem and then verified this fix:
> > > Simply create a single component yaml connection to Cosmos DB in folder `./components`, then run:
> > > ```shell
> > > for i in {1..50}
> > > do
> > >   dapr run --log-level warn --components-path components&
> > >   echo "Sidecar $i"
> > > done
> > > dapr run --log-level warn --components-path components
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > You will observe that after about 10 side cars additional sidecars fail to connect due to exceeding CosmosDB request limits.
> > > Using a custom build of `daprd` which includes this PR I was able to connect all `50` sidecars just fine in about 5 seconds.
> > > Important: You must set a larger `initTimeout` in your component yaml for Cosmos DB.
> > > Personally I added `initTimeout: 5m` because the maximum total retry interval is 5 minutes.
> > 
> > 
> > Please add those useful info to docs.
> 
> +1

Yup, I'll be writing the docs later but it's on my to do list for this release.

Reply:
@pkedy no constant is exposed.

@daixiang0 this is a standard HTTP error code.

Where do you want to define standard HTTP codes?

Reply:
HTTP 429 means "Too Many Requests". The function is even called "isTooManyRequestsError"

Reply:
This is what the http package has:

```golang
StatusTooManyRequests = 429 // RFC 6585, 4
```

I wouldn't import `http` just for this, just define as it as an const in the file.

Reply:
This str and not int BTW.